internal/graphql/resolver/error.go

```diff
@@ -7,8 +7,10 @@ import (
 const (
 	errCode               = "test_code"
 	ErrInvalidRequestCode = "test_code"
+	ErrNilClientCode      = "test_code"
```

Comment:
Have we made enough initial decisions on the messaging framework to start assigning error codes?

Reply:
Not yet, we will need to start writing the script for autogeneration as an initial step. Before that we shud stick to the current practice.